casepro/msgs/management/commands/export_messages.py

```diff
@@ -0,0 +1,29 @@
+from __future__ import absolute_import, unicode_literals
+
+import json
+from dash.orgs.models import Org
+from django.core.management.base import BaseCommand, CommandError
+
+
+class Command(BaseCommand):
+    help = 'Exports all outbound messages with their matching original inbound message as JSON'
+
+    def add_arguments(self, parser):
+        parser.add_argument('org_id', type=int, metavar='ORG', help="The org to pull messages for")
+
+    def handle(self, *args, **options):
+        org_id = int(options['org_id'])
+        try:
+            org = Org.objects.get(pk=org_id)
+        except Org.DoesNotExist:
+            raise CommandError("No such org with id %d" % org_id)
+
+        for message in org.outgoing_messages.filter(reply_to__isnull=False):
```

Comment:
Maybe a `select_related`  on `reply_to` so that we don't have to do an additional db lookup for every message?